frontend/src/App.js

```diff
@@ -6,7 +6,7 @@ import SignUp from './components/SignUp/SignUp.js';
 import UsersAll from './components/AdminControls/UsersAll.js';
 import User from './components/AdminControls/User.js';
 import history from './history.js';
-import RecipeList from './components/recipeCard/recipeCard.js';
+import RecipeList from './components/RecipeCard/recipeCard.js';
```

Comment:
I think the recipeCard.js should have a capital R?

Reply:
The app is compiling just fine and I've triple checked all the naming/capitalization so maybe best to ignore circleCi check?

Reply:
changing line 9 to ```import RecipeList from './components/recipeCard/recipeCard.js';``` should resolve the circleCi issue